config/config_helpers_test.go

```diff
@@ -138,11 +138,17 @@ func TestRunCommand(t *testing.T) {
 		expectedErr       error
 	}{
 		{
-			`"/bin/bash", "-c", ""echo -n foo""`,
+			`"/bin/bash", "-c", "echo -n foo"`,
 			terragruntOptionsForTest(t, homeDir),
 			"foo",
 			nil,
 		},
+		{
+			`"/bin/bash", "-c", "FOO=\"a test\"; echo -n $FOO"`,
```

Comment:
A simple test that verifies handling of escaped quotes.  I didn't want to write an unit test that relies on `jq`.